docs/install.md

```diff
@@ -936,3 +936,155 @@ This procedure can be used to downgrade the active version of SAT.
     the modified CFS configuration. If more than one CFS configuration was
     modified, use the first one.
 
+## Optional: Configuring SAT to run on an external system over the CAN
+
+It is possible to configure SAT so that it can interact with CSM over the CAN. This allows SAT to run from a physical or
+virtual machine which is not a part of the CSM management cluster.
+
+### Limitations
+
+Some SAT subcommands must interact with services which run on the NCNs in the CSM management cluster, including CSM REST
+APIs, Kubernetes, Ceph, and S3. Certain SAT commands, such as `sat bootsys`, may not work as expected.
+
+Instructions for installing SAT on an external system are provided "as-is", and installing SAT on an external system is
+not an officially supported configuration. These instructions are provided with the hope that they can useful for users
+who desire additional flexibility.
+
+Certain additional steps may need to be taken to install and configure SAT depending on the configuration of the
+external system in use. These additional steps may include provisioning virtual machines, installing packages, or
+configuring TLS certificates, and these steps are outside the scope of this documentation. This section covers only the
+steps needed to configure SAT to use externally-accessible API endpoints exposed by CSM.
+
+### Prerequisites
+
+- The external system must be on the Customer Access Network (CAN).
+- The SAT Python package and its dependencies, or the SAT podman image and wrapper script, have been installed on the
+  external system.
+- `kubectl` is installed on the external system.
+- The root CA certificates used when installing CSM have been added to the external system's trust store such that
+  authenticated TLS connections can be made to the CSM REST API gateway. See the [Certificate
+  Authority](https://cray-hpe.github.io/docs-csm/en-12/background/certificate_authority/) section of the CSM
+  documentation for more information.
+
+### Procedure
+
+1. Copy the file `/etc/kubernetes/admin.conf` from `ncn-m001` to `~/.kube/config` on the external system.
```

Comment:
For consistency with other numbered lists, add an extra space after the number before the text so all lines in a numbered list are indented 4 chars.

```suggestion
1.  Copy the file `/etc/kubernetes/admin.conf` from `ncn-m001` to `~/.kube/config` on the external system.
```